docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: "3"
 services:
 
   development:
+    platform: linux/x86_64
```

Comment:
is this supported in the version 3 docker-compose syntax? Also, wouldn't it be the docker container that is platform specific not the compose file?

Reply:
It's supported, and ya it's just that container.